src/blocks/net.rs

```diff
@@ -855,6 +857,8 @@ impl Block for Net {
         let is_up = self.device.is_up()?;
         if !exists || !is_up {
             self.active = false;
+            self.exists = exists;
```

Comment:
Maybe better rewritten as:
```rust
let self.exists = self.device.exists()?;
let self.active = self.device.is_up()?;
if !self.exists || !self.is_up {
...
```
?

Reply:
Alright, i did it a little differently than you suggested because it's even cleaner to clarify that `inactive` includes `missing`